Dispatchfile

```diff
@@ -70,18 +70,59 @@ dindTask("lint-yaml",
                  workingDir="/workspace/{git_clone_dirname}/".format(git_clone_dirname=git_clone_dirname),
                  command=["yamllint", "--config-file", "test/yamllint.yaml", "."],
              ),
-         ]
-         )
+         ])
+
+dindTask("install-job",
+         inputs=[git_clone_dirname],
+         deps=[kind_cluster],
+         steps=[
+             fetch_kubeconfig(kind_cluster_name, git_clone_dirname),
+
+             k8s.corev1.Container(
+                 name="fetch-master",
+                 workingDir="/workspace/{git_clone_dirname}/".format(git_clone_dirname=git_clone_dirname),
+                 args=["git", "fetch", "origin", "master"]),
+
+             k8s.corev1.Container(
+                 name="install-job",
+                 image="mesosphere/kubeaddons-ci:dispatch@sha256:80a6f3c8775133b4ad2fab3d44ab4fbb4b541dbc03fcdb296b13f67611427d8c",
+                 workingDir="/workspace/{git_clone_dirname}/".format(git_clone_dirname=git_clone_dirname),
+                 command=["/entrypoint.sh", "make", "dispatch-test-install-upgrade"],
+
+                 env= dispatch_common_go_environment +[
```

Comment:
what is dispatch_common_go_environment

Reply:
removed, that was my attempt to fix the dispatch issues, what did change is me adding a few more environment variables. It seems that it doesnt like that?